### DISCUSSION TITLE
Symfony Flex Structure

### DISCUSSION THREAD

**Discussion by Maxara** [pimcore/pimcore#7895] [Q&A / Support]
Hello all,
symfony flex structure has finally arrived the pimcore demo :smile:  Would it be a good idea to refactor the folder structure of a new installed pimcore project also this way (for beeing prepared to the pimcore x update)? Or is there even the plan from pimcore to adjust the pimcore installer concerning symfony flex structure?

Thank you in advance

**Comment by Maxara** [comment]
Nobody any idea? @dvesh3 maybe you can share your experience concerning the refactoring to Symfony Flex structure? We are starting a new Pimcore Project the next weeks and I am not sure whether it makes sense to start right from the beginning with the flex folder structure to be better prepared for the upcoming pimcore x update.

**Reply by dpfaffenbauer** [reply]
Yeah no... stick to what is preferred in Pimcore 6 and move the structure once you update to X. It is quite easy to migrate the structure.

**Reply by dpfaffenbauer** [reply]
or be brave and start with X ;)

**Reply by Maxara** [reply]
Hmm, but at latest with pimcore x it is mandatory to use the flex structure?

And let me think about beeing brave ... ahm no, I think I am not :smile:
